backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -309,30 +312,44 @@ protected void runLiquibaseMigration(DatabaseConfig config, String liquibaseRoot
   }
 
   protected static void createDBIfNotExists(RdbConfig rdb) throws SQLException {
-    LOGGER.info("Checking DB: {}", rdb.getRdbUrl());
     var properties = new Properties();
     properties.put("user", rdb.getRdbUsername());
     properties.put("password", rdb.getRdbPassword());
-    properties.put("sslMode", rdb.getSslMode());
-    final var dbUrl = RdbConfig.buildDatabaseServerConnectionString(rdb);
+
+    if (rdb.getDBConnectionURL() == null) {
+      properties.put("sslMode", rdb.getSslMode());
+    }
+
+    var dbUrl = RdbConfig.buildDatabaseServerConnectionString(rdb);
     LOGGER.info("Connecting to DB server url: {} ", dbUrl);
+
+    if (rdb.isMssql()) {
+      Pattern pattern = Pattern.compile("jdbc:sqlserver://([^;]*)", Pattern.CASE_INSENSITIVE);
```

Comment:
The regex links in comments used in the other file below are very useful; it would be great to have a similar one here as well.